server/startup/migrations/v148.js

```diff
@@ -0,0 +1,24 @@
+import { Migrations } from '../../../app/migrations/server';
+import { settings } from '../../../app/settings';
+import { Settings } from '../../../app/models';
+
+Migrations.add({
+	version: 148,
+	up() {
+		settings.get('Accounts_OAuth_Gitlab', function(key, value) {
```

Comment:
please don't use a callback here. it will register a callback to be run every time the setting changes